### PR TITLE
Case count line graph

### LINKED ISSUE #133: write dashboard "case count over doc date line graph" script
## Task Description

- 

## User Stories

- As a user of the dashboard, I want to be able to see, for certain judge gender, verdict, etc. settings, how many cases there were and when they were documented.

## Relevant Files [If Available] 

-

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
Also changed the judges line graph to include 0 counts.

Closes #133

### CODE REVIEW

**Comment by luciegamby-sigmalabs** [comment]
Resolving conflicts now

File: dashboard/charts.py [hunk 1]
```diff
@@ -82,6 +82,20 @@ def get_judges_appointed(conn: connect) -> pd.DataFrame:
     return pd.DataFrame(rows)
 
 
+def get_cases(conn: connect) -> pd.DataFrame:
+    '''Returns the case number and date.'''
+
+    query = """
+                SELECT COUNT(case_no), transcript_date FROM transcript
+                GROUP BY transcript_date;
+                """
```
Comment (MahinRahman8901) [inline dashboard/charts.py:91]: really smart query

File: dashboard/charts.py [hunk 2]
```diff
@@ -135,7 +149,25 @@ def get_judge_count_line_chart(conn: connect) -> alt.Chart:
...
+
+        if not ((judge_count['gender'] == "F") & (judge_count['appointed'] == pd.Timestamp(year=i, month=12, day=31))).any():
+            row = pd.DataFrame({"gender": "F", "appointed": pd.Timestamp(
+                year=i, month=12, day=31), "count": 0}, index=[0])
+            judge_count = pd.concat([judge_count, row], ignore_index=True)
+
+        if not ((judge_count['gender'] == "X") & (judge_count['appointed'] == pd.Timestamp(year=i, month=12, day=31))).any():
+            row = pd.DataFrame({"gender": "X", "appointed": pd.Timestamp(
+                year=i, month=12, day=31), "count": 0}, index=[0])
+            judge_count = pd.concat([judge_count, row], ignore_index=True)
```
Comment (MahinRahman8901) [inline dashboard/charts.py:170]: Nice work on catching these edge cases!

File: dashboard/charts.py [hunk 3]
```diff
@@ -146,6 +178,23 @@ def get_judge_count_line_chart(conn: connect) -> alt.Chart:
...
+    data["transcript_date"] = pd.to_datetime(data["transcript_date"])
+    case_count = data.set_index("transcript_date")
+
+    case_count = case_count.groupby(
+        [pd.Grouper(freq='YE')]).count().reset_index()
+
+    chart = alt.Chart(case_count, title="Case Count / Time").mark_line().encode(
+        x=alt.X("transcript_date", title="Month of Case"),
+        y=alt.Y("count", title="Number of Cases"),
+    )
```
Comment (MahinRahman8901) [inline dashboard/charts.py:195]: Nice chart code

**Review by MahinRahman8901 — APPROVED** [review]

**Pull request merged**